### ISSUE TITLE
AttributeError: 'EntryBreakBreakpoint' object has no attribute 'locations'

### ISSUE THREAD

**Issue by cnitlrt** [bata24/gef#68]
Hello, when i debug a binary ,I found a bug
```
AttributeError: 'EntryBreakBreakpoint' object has no attribute 'locations'
----------------------------- Detailed stacktrace ------------------------------
File "/home/ubuntu/geff/gef.py", line 24863, in get_breakpoints()
    ->     for bl in b.locations:
File "/home/ubuntu/geff/gef.py", line 24880, in context_code()
    ->     bp_locations = self.get_breakpoints()
File "/home/ubuntu/geff/gef.py", line 24649, in do_invoke()
    ->     self.layout_mapping[section]()
File "/home/ubuntu/geff/gef.py", line 10679, in wrapper()
    ->     return f(*args, **kwargs)
File "/home/ubuntu/geff/gef.py", line 10649, in wrapper()
    ->     return f(self, args, **kwargs)
File "/home/ubuntu/geff/gef.py", line 13083, in invoke()
    ->     self.do_invoke(argv)
```
this is my enviroment
```
OS:                     Ubuntu 20.04.6 LTS
kernel (uname -a):      Linux ubuntu-BQM5 5.15.0-67-generic #74~20.04.1-Ubuntu SMP Wed Feb 22 14:52:34 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
kernel (/proc/version): Linux version 5.15.0-67-generic (buildd@lcy02-amd64-029) (gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0, GNU ld (GNU Binutils for Ubuntu) 2.34) #74~20.04.1-Ubuntu SMP Wed Feb 22 14:52:34 UTC 2023
GEF:                    Last modified: 2024-03-26 04:18:26 SHA1: 8d4ba44a95c13bf27761ebdbfcc21c018d87c6e4
gdb:                    9.2
python:                 3.8.10 (default, Nov 22 2023, 10:22:35)  [GCC 9.4.0]
capstone:               5.0.1280
keystone:               0.9.9
unicorn:                2.0.1
ropper:                 1.13.8
gcc:                    gcc (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0
readelf:                GNU readelf (GNU Binutils for Ubuntu) 2.34
objdump:                GNU objdump (GNU Binutils for Ubuntu) 2.34
seccomp-tools:          SeccompTools Version 1.6.1
one_gadget:             OneGadget Version 1.9.0
```

**Comment by bata24** [comment]
Thank you for the report.

It seems that it was degraded with f81b13136a3073874c30c04e5ae6d9c29132e2ce.
In gdb 9.x, breakpoint objects did not have a `locations` member.
I fixed it (see 80923ec7425b536c517b86dececec0f20aeb8f7e), so could you please try the latest version?

**Comment by cnitlrt** [comment]
Thank you for your prompt response. I've had a chance to test it, and it's working flawlessly. I genuinely appreciate your efforts in both creating and maintaining such an excellent extension. Given the resolution, I will now proceed to close this issue
